### DISCUSSION TITLE
Enabled Masking Password feature

### DISCUSSION THREAD

**Discussion by jairogutierrezpayu** [arkmq-org/arkmq-org-broker-operator#454] [Q&A] [answered]
Hi, how i can enabled 'masking password' feature using operator?

https://activemq.apache.org/components/artemis/documentation/latest/masking-passwords

Thx!

**Comment by jairogutierrezpayu** [comment, marked as answer]
I can enabled this feature using ENC(passwordcoded) syntax in secret creation, this case use default codec.

**Reply by brusdev** [reply]
@jairogutierrezpayu can I mark your reply as answer?

**Reply by jairogutierrezpayu** [reply]
sure, thx

**Reply by brusdev** [reply]
thanks for your answer, it will help other users.
